src/main/java/com/baidu/hugegraph/structure/Element.java

```diff
@@ -34,7 +34,10 @@ public int hashCode() {
 
     @Override
     public boolean equals(Object other) {
-        if (!(other instanceof Element)) {
+        if (this == other) {
+            return true;
```

Comment:
not covered by tests

Reply:
There is no case test to this line for the time being

Reply:
If we are pursuing coverage, we can write an elementTest, it doesn’t feel necessary for the time being